modules/snooker/src/main/ts/ephox/snooker/api/Warehouse.ts

```diff
@@ -100,11 +123,28 @@ const justCells = function (warehouse: Warehouse) {
   return Arr.flatten(rows);
 };
 
+const justColumns = (warehouse: Warehouse) =>
+  Arr.map(warehouse.columns, (group) => group);
+
+const hasColumns = (warehouse: Warehouse) =>
+  warehouse.columns.length > 0;
+
+const getColumnAtIndex = (warehouse: Warehouse, columnIndex: number) => {
```

Comment:
This is an inconsistent name... it should probably be `getColumn` or `getColumnAt`.